sdk/ai/azure-ai-projects/README.md

```diff
@@ -571,6 +571,108 @@ print(f"Created agent, agent ID: {agent.id}")
 
 <!-- END SNIPPET -->
 
+To make a function call we need to create and deploy the Azure function. In the code snippet below, we have an example of function on C# which can be used by the code above.
```

Comment:
Seems odd to have a C# Azure function example if a README.md of a Python package. Is there a way to run Python code as Azure function, and if so, should we not show that here instead?

Reply:
Replaced C# function by python.